docs/.vitepress/ex-base.js

```diff
@@ -0,0 +1,55 @@
+import { createApp } from 'vue';
+
+export const buildWc = (elementName, baseVueComponent) => {
+  if (typeof window === 'undefined') return;
+  if (!customElements.get(elementName))
+    customElements.define(
+      elementName,
+      class extends HTMLElement {
+        connectedCallback() {
+          const warp = `<link rel="stylesheet" type="text/css" href='https://assets.finn.no/pkg/@warp-ds/tokens/v1/finn-no.css' />`;
+          const target = `<div id="app" class="mt-16"></div>`;
+          const shadowUnoStyle = `<style>
+          @unocss-placeholder
+        h3 {
+          font-weight: bold;
+        }
+        .component {
+          padding: 16px;
+          border: 3px solid rgb(246, 248, 250);
+          margin-bottom: 8px;
+        }
+        .options {
+          margin-bottom: 8px;
+
+        }
+        .options span {
+          margin-right: 12px;
+          margin-left: 4px;
+        }
+      </style>`;
+
+          this.shadow = this.attachShadow({ mode: 'open' });
+          this.shadow.innerHTML = shadowUnoStyle + warp + target;
```

Comment:
I think warp should actually come first because @unocss-placeholder that is added to `shadowUnoStyle` relies on some tokens from the brand css. On another note, I **think** that the brand css file can live in the document head only if we're not adding theme-switching logic here. Everything it contains should penetrate shadow DOM.